charts/longhorn/102.4.0+up1.6.1/values.yaml

```diff
@@ -0,0 +1,483 @@
+      repository: rancher/mirrored-longhornio-longhorn-instance-manager
+      # -- Specify Longhorn instance manager image tag
+      tag: v1.6.1
+    shareManager:
+      # -- Repository for the Longhorn Share Manager image.
+      repository: rancher/mirrored-longhornio-longhorn-share-manager
+      # -- Specify Longhorn share manager image tag
+      tag: v1.6.1
+    backingImageManager:
+      # -- Repository for the Backing Image Manager image. When unspecified, Longhorn uses the default value.
+      repository: rancher/mirrored-longhornio-backing-image-manager
+      # -- Specify Longhorn backing image manager image tag
+      tag: v1.6.1
+    supportBundleKit:
+      # -- Repository for the Longhorn Support Bundle Manager image.
+      repository: rancher/mirrored-longhornio-support-bundle-kit
+      # -- Tag for the Longhorn Support Bundle Manager image.
+      tag: v0.0.36
+  csi:
+    attacher:
+      # -- Repository for the CSI attacher image. When unspecified, Longhorn uses the default value.
+      repository: rancher/mirrored-longhornio-csi-attacher
+      # -- Tag for the CSI attacher image. When unspecified, Longhorn uses the default value.
+      tag: v4.4.2
+    provisioner:
+      # -- Repository for the CSI Provisioner image. When unspecified, Longhorn uses the default value.
+      repository: rancher/mirrored-longhornio-csi-provisioner
+      # -- Tag for the CSI Provisioner image. When unspecified, Longhorn uses the default value.
+      tag: v3.6.2
+    nodeDriverRegistrar:
+      # -- Repository for the CSI Node Driver Registrar image. When unspecified, Longhorn uses the default value.
+      repository: rancher/mirrored-longhornio-csi-node-driver-registrar
+      # -- Tag for the CSI Node Driver Registrar image. When unspecified, Longhorn uses the default value.
+      tag: v2.9.2
+    resizer:
+      # -- Repository for the CSI Resizer image. When unspecified, Longhorn uses the default value.
+      repository: rancher/mirrored-longhornio-csi-resizer
+      # -- Tag for the CSI Resizer image. When unspecified, Longhorn uses the default value.
+      tag: v1.9.2
+    snapshotter:
+      # -- Repository for the CSI Snapshotter image. When unspecified, Longhorn uses the default value.
+      repository: rancher/mirrored-longhornio-csi-snapshotter
+      # -- Tag for the CSI Snapshotter image. When unspecified, Longhorn uses the default value.
+      tag: v6.3.2
+    livenessProbe:
+      # -- Repository for the CSI liveness probe image. When unspecified, Longhorn uses the default value.
+      repository: rancher/mirrored-longhornio-livenessprobe
+      # -- Tag for the CSI liveness probe image. When unspecified, Longhorn uses the default value.
+      tag: v2.12.0
+  openshift:
+    oauthProxy:
+      # -- Repository for the OAuth Proxy image. This setting applies only to OpenShift users.
+      repository: rancher/mirrored-longhornio-openshift-origin-oauth-proxy
```

Comment:
Hi @mantissahz we need to add this mirror image to questions.yaml too. Could you help to update the PR?

Reply:
Added. Thanks